src/react/src/util/util.js

```diff
@@ -1155,10 +1155,10 @@ function descendingComparator(a, b, orderBy) {
         bValue = bValue.toLowerCase();
     }
 
-    if (bValue < aValue) {
+    if (aValue === null || bValue < aValue) {
```

Comment:
Why this null check is needed here? Is it fix some edge case?

Reply:
@VadimKovalenkoSNF  We have requirements that empty/NULL values should be on top if we sort from recent to oldest & on bottom if we sort from oldest to recent. This case handled here.

Reply:
Got it, thank you!